dashboard/lib/tasks/seed.rake

```diff
@@ -150,6 +150,69 @@ namespace :seed do
     oceans
     sports
   ).map {|script| "config/scripts_json/#{script}.script_json"}.freeze
+
+  ADHOC_TEST_SCRIPTS = UI_TEST_SCRIPTS + %w(
```

Comment:
I would suggest omitting `UI_TEST_SCRIPTS` here, otherwise this PR could add substantial time to adhoc startup.